pkg/connector/msgconv.go

```diff
@@ -38,19 +38,19 @@ import (
 	"go.mau.fi/mautrix-twitter/pkg/twittermeow/data/types"
 )
 
-func (tc *TwitterClient) convertEditToMatrix(ctx context.Context, portal *bridgev2.Portal, intent bridgev2.MatrixAPI, existing []*database.Message, data *types.MessageData) (*bridgev2.ConvertedEdit, error) {
+func (tc *TwitterConnector) convertEditToMatrix(ctx context.Context, portal *bridgev2.Portal, intent bridgev2.MatrixAPI, cli *twittermeow.Client, existing []*database.Message, data *types.MessageData) (*bridgev2.ConvertedEdit, error) {
```

Comment:
This seems like an unnecessary change, TwitterClient can already access the connector